modules/spotxBidAdapter.js

```diff
@@ -501,15 +497,11 @@ function outstreamRender(bid) {
         if (!framedoc && rawframe.contentWindow) {
           framedoc = rawframe.contentWindow.document;
         }
-        framedoc.body.appendChild(script);
+        script = loadExternalScript(`https://js.spotx.tv/easi/v1/${bid.channel_id}.js`, 'outstream', undefined, framedoc);
       } else {
-        const slot = getBidIdParameter('slot', bid.renderer.config.outstream_options);
-        if (slot && window.document.getElementById(slot)) {
-          window.document.getElementById(slot).appendChild(script);
-        } else {
-          window.document.getElementsByTagName('head')[0].appendChild(script);
-        }
+        script = loadExternalScript(`https://js.spotx.tv/easi/v1/${bid.channel_id}.js`, 'outstream');
```

Comment:
`loadExternalScript` does not support this use case of appending under the slot element.

I do not know the use case enough to know if it matters so asking for clarification here.

Reply:
The "slot" use case is used when the publisher wants to load the ad in a specific element that already exists on the page. If we just insert the easi script in the head it will create a new element and render the video in that newly created element. This would have the effect of not allowing the publisher to place the ad in the correct place on the page.

Reply:
@spotxslagle Can you confirm this is still the case. It has been a few years since I last looked at these integrations so I may be misremembering some of the configuration details.

Reply:
Perhaps missed showheroes, however, why not specify the script as a renderer rather than doing the append child?